src/main/java/TaskManager.java

```diff
@@ -0,0 +1,86 @@
+import java.util.ArrayList;
+public class TaskManager {
+    private static final ArrayList<Task> Tasks = new ArrayList<Task>();
```

Comment:
The use of ArrayList object is good! This helps to eliminate problem such as magic numbers or the problem of worrying the size of the list